documentation/staging/content/userguide/managing-operators/installation.md

```diff
@@ -155,6 +163,9 @@ see [Troubleshooting]({{<relref "/userguide/managing-operators/troubleshooting#c
   A backslash (`\`) is only required when specifying the selector on the command line using `--set`,
   as shown in the previous example.
 
+### Install the WebLogic domain resource conversion webhook
+Refer to [install the conversion webhook]({{<relref "/userguide/managing-operators/conversion-webhook#install-the-conversion-webhook" >}}) section for the conversion webhook installation details. 
```

Comment:
Consider adding a little more detail:

By default, the WebLogic domain resource conversion webhook is automatically installed the first time an operator is installed in a cluster, and removed the first time an operator is uninstalled. If you are using multiple operators, or want to be able to create or alter domains even when no operator's are running, then you will need to fine tune this life-cycle.  Refer to ...